service/policy/subjectmapping/subject_mapping.go

```diff
@@ -37,7 +42,7 @@ func (s SubjectMappingService) CreateSubjectMapping(ctx context.Context,
 	rsp := &sm.CreateSubjectMappingResponse{}
 	slog.Debug("creating subject mapping")
 
-	sm, err := s.dbClient.CreateSubjectMapping(context.Background(), req)
+	sm, err := s.dbClient.CreateSubjectMapping(ctx, req)
```

Comment:
Just want to flag this as it was flagged with lint and I changed the line to use the context passed as an argument, but want to make sure that's what we want here

Reply:
yes generally you will either pass the same context or a lightly modified version of the context. This allows [functions to be cancelable](https://go.dev/blog/context), a desirable property especially when they call out to other services e.g. a REST endpoint, RPC or even just perform file i/o on large files.